bearRouter.js

```diff
@@ -0,0 +1,118 @@
+const router = require("express").Router();
+const knex = require("knex");
+
+const knexConfig = {
+  client: "sqlite3",
+  useNullAsDefault: true,
+  connection: {
+    filename: "./data/lambda.sqlite3"
+  }
+};
```

Comment:
this is structured properly, but notice how you have to declare this in both of your router files -- not very DRY. If you extract this into a dbConfig file (or name of your choice) and export it, you can import the reference in one line in both router files instead